### ISSUE TITLE
documentation says `--version` is supported

### ISSUE THREAD

**Issue by SethTisue** [olafurpg/scalafmt#121]
manual says:

```
scalafmt --version // should be 0.1.2
```

but:

```
% brew install scalafmt
==> Installing scalafmt from olafurpg/scalafmt
==> Downloading https://github.com/olafurpg/scalafmt/releases/download/v0.1.2/scalafmt.tar.gz
==> Downloading from https://github-cloud.s3.amazonaws.com/releases/45423600/b1489fec-e6d3-11e5-8a7e-6d642d7a6e00.gz?X-A
######################################################################## 100.0%
==> ./bin/configure
🍺  /usr/local/Cellar/scalafmt/0.1.2: 3 files, 36.8M, built in 11 seconds
% scalafmt --version
Error: Unknown option --version
Try --help for more information.
```

**Comment by olafurpg** [comment]
For some reason I assumed scopt would include `--version` by default. Have a fix for 0.1.3.

**Comment by olafurpg** [comment]
Fixed in #124
